<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ntalk</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/th.png</normaloff>:/th.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#Login{
	background-color: rgb(253, 211, 68);
}

QLineEdit{

	color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton{

border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{

border-style:inset;
font-style:MingLiU-ExtB;
}

QComboBox{
	color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font-size:16px;

border-radius:10px;
font-style:MingLiU-ExtB;
}

QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">LineEdit{
color:#8d98a1;
background-color:#405361;
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:groove;
border-color: rgb(170, 170, 255);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#loginButton:!hover{
borde-radius:10px;
background-color:rgb(255,255,0);
}
#loginButton:hover{
borde-radius:10px;
background-color:rgb(255,255,255);
}</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#registerButton:!hover{
borde-radius:10px;
background-color:rgb(255,128,0);
}
#registerButton:hover{
borde-radius:10px;
background-color:rgb(255,83,83);
}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>675</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>loginButton</zorder>
  <zorder>registerButton</zorder>
  <zorder>checkBox</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
